bundles/core/org.eclipse.smarthome.core.thing/ESH-INF/i18n/DefaultSystemChannels_fr.properties

```diff
@@ -4,3 +4,27 @@ channel-type.system.battery-level.label = Niveau de la batterie
 channel-type.system.trigger.label = D�clencheur
 channel-type.system.rawbutton.label = Bouton simple
 channel-type.system.button.label = Bouton
+// channel-type.system.rawrocker.label = 
```

Comment:
What about this one?

Reply:
I have absolutely no idea what it represents and what it is for !! If at least there was a clear description of the channel...
I even tried to translate in french but with no success.

Reply:
As I don't word if it is about the meaning or about finding the right words, here the meaning:

Does this one helps: https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html

A button is something that can be pressed, a switch (e.g. light) can be seen as two buttons, one up and one down.

Reply:
Ok, it is probably "bouton à bascule" in French...